pyrfc/_pyrfc.pyx

```diff
@@ -2051,19 +2051,20 @@ cdef wrapString(SAP_UC* uc, uclen=-1, rstrip=False):
     if uclen == -1:
         uclen = strlenU(uc)
     if uclen == 0:
-        return ''
-    cdef unsigned utf8_size = uclen * 3 + 1
+        return ''
+    cdef unsigned utf8_size = uclen * 5 + 1

```

Comment:
This allows for longer unicode characters - but I'm pretty sure is not the cause of the problem here